pkg/cmd/root/help_reference.go

```diff
@@ -0,0 +1,82 @@
+package root
```

Comment:
Now that we have more than one help topic, does it make sense to move them into their own package?

Reply:
i'm ok with kicking that can a little further down the road.